tests/test_formsearch.py

```diff
@@ -30,8 +30,8 @@
 </form>
 <h1>Login</h1>
 <form action="/log_me_in">
-<input name="user_field">
-<input name="password_field">
+<input name="User_Field">
+<input name="Password_Field">
```

Comment:
this worries me. why do we have to change the test corpus to accomodate this change?

Reply:
There was quite a long discussion from https://github.com/linkchecker/linkchecker/pull/360#pullrequestreview-391371004 to nearly the end of that PR (this test is new and only introduced by #358), which concluded that being case sensitive was the way to go.